llvm/lib/Target/ARM/MCTargetDesc/ARMInstPrinter.cpp

```diff
@@ -372,7 +372,7 @@ void ARMInstPrinter::printThumbLdrLabelOperand(const MCInst *MI, unsigned OpNum,
     return;
   }
 
-  O << markup("<mem:") << "[pc, ";
+  O << markup("<mem:") << "[" << markup("<reg:") << "pc" << markup(">") << ", ";
```

Comment:
How do we use `<mem`?
I guess we only consider the projection of  the topmost tag LLVM gives us?

So this is like `<mem:[><reg:pc><mem:, ...>`?

Reply:
Yep, we sort the tags explicitly (innermost first) before "flattening" them. See the relevant commit in the revng-counterpart.